pandas/tests/groupby/transform/test_transform.py

```diff
@@ -1511,10 +1511,5 @@ def test_null_group_str_transformer_series(request, dropna, transformation_func)
     msg = f"{transformation_func} is deprecated"
     with tm.assert_produces_warning(warn, match=msg):
         result = gb.transform(transformation_func, *args)
-    if dropna and transformation_func == "fillna":
-        # GH#46369 - result name is the group; remove this block when fixed.
-        tm.assert_equal(result, expected, check_names=False)
-        # This should be None
-        assert result.name == 1.0
-    else:
-        tm.assert_equal(result, expected)
```

Comment:
can u link this PR to the referenced issue as it appears to close it as well

Reply:
Actually #46369 is already the issue being closed